src/components/Header/Header.test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
According the "guiding principles" of the React Testing Library, it's best to avoid selecting elements based on class names or dom structure, seen here:
>That said, they are way better than querying based on DOM structure or styling css class names. 
- https://testing-library.com/docs/dom-testing-library/api-queries

The main functionality I think is worth testing for the header is the following:
1. it renders a title
2. It renders a menu
3. For each menu item it renders a link to the provided path